src/action.ts

```diff
@@ -78,50 +78,47 @@ export const runAction = async (space): Promise<void> => {
     },
   };
 
-  if (CREATE_CDA_TOKEN) {
-    core.startGroup(`Ephemeral CDA token creation`)
-
-    const branchName = branchNames.headRef;
-    const keyName = `ephemeral-token-${branchName}`;
+  const branchName = branchNames.headRef;
+  const tokenKeyName = `ephemeral-token-${branchName}`;
 
+  if (CREATE_CDA_TOKEN) {
     const spaceKeys = await space.getApiKeys();
 
-    const exists = spaceKeys.items.some(item => item.name === keyName)
+    const exists = spaceKeys.items.some(item => item.name === tokenKeyName)
 
     if (exists) {
-      Logger.log(`CDA token ${keyName} is already created`);
+      Logger.log(`CDA token ${tokenKeyName} is already created`);
     } else {
-      Logger.log(`Creating new CDA token "${keyName}" for ephemeral environment "${environmentId}"...`);
+      Logger.log(`Creating new CDA token "${tokenKeyName}" for ephemeral environment "${environmentId}"...`);
 
       try {
         const key = await space.createApiKey({
-          name: keyName,
+          name: tokenKeyName,
           environments: [newEnv],
         })
 
-        // set token as secret just in case
-        core.setSecret(key.accessToken)
-      
-        core.setOutput(
-          "cda_token",
-          key.accessToken
-        );
         Logger.success("CDA token has been created");
       } catch(err) {
         Logger.warn("unable to create ephemeral token");
         Logger.verbose(err)
       }
     }
-
-    core.endGroup()
   }
 
   Logger.verbose("Update API Keys to allow access to new environment");
 
   const { items: keys } = await space.getApiKeys();
   await Promise.all(
     keys.map((key) => {
-      Logger.verbose(`Updating: "${key.sys.id}"`);
+      // put token value on every action run
+      // helpful in case the first run failed and the "Write Comment" step was not reached
+      if (key.name == tokenKeyName) {
```

Comment:
Let's plz use ===

Reply:
I'll correct it in my current PR
- https://github.com/the-infatuation/contentful-action/pull/11


thank you!